spacy/default_config.cfg

```diff
@@ -2,13 +2,15 @@
 lang = null
 stop_words = []
 lex_attr_getters = {}
+vocab_data = {}
```

Comment:
I'm wondering whether it would make sense to separate `nlp.vocab` section here?

Reply:
I was thinking about this too, yes! For now, I tried to avoid adding more subsections and I wasn't sure what should go under `vocab` (as opposed to the top level `nlp` – like, is `stop_words` vocab? Or `lex_attr_getters`?)

Reply:
Yeah, there's not other than meta much at the top level if you do this, I guess. Maybe this could just be `data` then? What non-vocab data is there potentially? (Could this just be `data` to be more like the other sections?)